GroupProject/Pages/Index.cshtml.cs

```diff
@@ -35,24 +35,21 @@ public void OnGet()
 
 
 
-                string Membership = webClient.DownloadString("https://api.sportsdata.io/v3/soccer/scores/json/MembershipsByCompetition/EPL?key=bc49021bad1943008414c5a75e665961");
+                string playerInfoData = webClient.DownloadString("https://api.sportsdata.io/v3/soccer/scores/json/MembershipsByCompetition/EPL?key=bc49021bad1943008414c5a75e665961");
                 JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("PlayerInfoSchema.json"));
-                JArray jarray = JArray.Parse(Membership);
+                JArray jArray = JArray.Parse(playerInfoData);
                 IList<string> validationEvents = new List<string>();
-                if (jarray.IsValid(schema, out validationEvents))
+                if (jArray.IsValid(schema, out validationEvents))
```

Comment:
Per best practices below, I changed “JArray jarray” to “JArray jArray”. Read that you should always use camel case when declaring variables

Source: https://www.codeproject.com/Articles/118853/Some-Best-Practices-for-C-Application-Developmen

Reply:
Ditto comments above.